src/show/pages/public/CreateProject/index.js

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios';
+import { useState } from 'react';
+import StepWizard from 'react-step-wizard';
+import { useNavigate } from 'react-router-dom';
+import { PROJECT_PATH } from '../../../../process/routes/paths';
+
+import {
+  FeeStep,
+  GoalStep,
+  TitleStep,
+  ImagesStep,
+  PublishStep,
+  CategoryStep,
+  DatelineStep,
+  StatementStep,
+  DescriptionStep,
+  TeamMembersStep
+} from '../CreateProject/Subviews';
+
+const CreateProject = () => {
+  const navigate = useNavigate();
+  const [project, setProject] = useState({
+    images: ['', '', '', '', '', '', '', ''],
+    team_members: [{}, {}, {}, {}, {}, {}, {}],
+    user_id: 1
+  });
+
+  const postData = e => {
+    const formData = new FormData();
+    project.images.forEach(image => {
+      if (image.length !== 0) {
+        formData.append('images[]', image);
+      }
+    });
+    formData.append('name', project.name);
+    formData.append('description', project.description);
+    formData.append('goal', project.goal);
+    formData.append('fee', project.fee);
+    formData.append('deadline', project.deadline);
+    project.team_members.forEach(member => {
+      if (Object.entries(member).length) {
+        formData.append('team_members_attributes[][name]', member.name);
+        formData.append(
+          'team_members_attributes[][avatar_url]',
+          member.avatar_url
+        );
+      }
+    });
+    formData.append('category', project.category);
+    formData.append('statement', project.statement);
+    formData.append('user_id', project.user_id);
+    e?.preventDefault();
```

Comment:
Move this prevent default up (to be the first line in this function)

Reply:
on top of the word